packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/WebViewClientHostApiImpl.java

```diff
@@ -63,12 +62,206 @@
     return requestData;
   }
 
+  @RequiresApi(Build.VERSION_CODES.N)
+  static class WebViewClientImpl extends WebViewClient implements Releasable {
+    private final Long instanceId;
+    private final InstanceManager instanceManager;
+    private final Boolean shouldOverrideUrlLoading;
+    private final WebViewClientFlutterApiImpl flutterApi;
+    private boolean ignoreCallbacks = false;
```

Comment:
Would it make sense for all of these wrappers to null out flutterApi instead of having a second variable, so that if someone forgets to check it's an immediate crash?

Reply:
I agree. I changed flutter api to be nullable and made it null in release